### PR TITLE
Replace webrick by puma

### PR DESCRIPTION
**Pull request by wiltonsr**
In konacha configuration file was made substitution of application server.

Signed-off-by: Wilton Rodrigues <braynwilton@gmail.com>
Signed-off-by: MatheusFerraz <matheus_ccmf@yahoo.com.br>

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -81,7 +81,7 @@ group :test do
   gem 'codeclimate-test-reporter', require: nil
 
   # For Konacha
-  gem 'thin'
+  gem 'puma'
```
Comment (diegoamc) [inline Gemfile]: I think we want puma to be used on all environments, not only `test`.

File: spec/models/kalibro_module_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe KalibroModule, :type => :model do
```
Comment (diegoamc) [inline spec/models/kalibro_module_spec.rb]: Is this part of the PR?

**Comment by diegoamc** [comment]
Is this PR meant to replace #387? If so, you are missing the changes on the `Procfile` too.

**Comment by wiltonsr** [comment]
Could you see what error is happening in Travis? This error doesn't occur in local environment when I execute the server, all test in `rake konacha:run` are passing.

**Comment by diegoamc** [comment]
The error message says:

> Capybara::Poltergeist::TimeoutError: Timed out waiting for response to {"name":"visit","args":["http://127.0.0.1:56741/"]}. It's possible that this happened because something took a very long time (for example a page load was slow). If so, setting the Poltergeist :timeout option to a higher value will help (see the docs for details). If increasing the timeout does not help, this is probably a bug in Poltergeist - please report it to the issue tracker.

Could you try increasing `Poltergeist` timeout and commit the changes?

**Comment by wiltonsr** [comment]
I've tried this. Setting from 20s to 60s. But it did not work

**Comment by wiltonsr** [comment]
ping

**Comment by rafamanzo** [comment]
I am able to reproduce the CI issues on my local environment today by running `rake spec && rake konacha:run`. It's a mystery why it fails only if I run unit tests before the JS. And a bigger mystery how CI fails without running the previously! haha

I've tried updating the gems for poltergeist (1.12.0), konacha (4.0.0) and puma (3.6.2) to no success.

I'll try replacing konacha by teaspoon. It have worked on other project I've worked when we had issues regarding Rails 5 compatibility. Not related at all, but may work.

**Comment by rafamanzo** [comment]
We are closing this PR in favor of #405, reopen if needed.

**Pull request closed without merging**